src/Controller/Component/PaginatorComponent.php

```diff
@@ -202,7 +203,8 @@ public function paginate($object, array $settings = [])
             'direction' => current($order),
             'limit' => $defaults['limit'] != $limit ? $limit : null,
             'sortDefault' => $sortDefault,
-            'directionDefault' => $directionDefault
+            'directionDefault' => $directionDefault,
+            'prefix' => Hash::get($options, 'prefix', null),
```

Comment:
Call to `Hash::get()` can be avoided by adding `'prefix' => null` to default options array on line 162.